### PR TITLE
Support comma-separated months

### PR DESCRIPTION
**Pull request by MerreM**
Hello,

Fair warning, this is my first pull request.

I'm using pretty cron and noticed it can't handle special characters yet.

For the project we're using it on, we don't need any others apart form commas (in months), so I thought I'd add support. 

There's a few tests for the changes and I've broken support for python 2,  as it won't affect me. 

I'll have a look at making it python 2 compatible as well if you want/need.

### CODE REVIEW

**Comment by adamchainz** [comment]
Thanks! I'll squash your commits and fix it up for Python 2 before merging.

File: pretty_cron/api.py [hunk 1]
```diff
@@ -65,6 +70,12 @@ def _pretty_date(month_day, month, week_day):
 
 
 def _human_month(month):
+    if type(month)==tuple:
```
Comment (adamchainz) [inline pretty_cron/api.py]: `type(x) == tuple` is not the greatest, `isinstance(x, tuple)` is preferable. I don't think it makes such a difference in this case, but it's a good habit.

**Comment by MerreM** [comment]
Switched the comparator. Cheers!

File: pretty_cron/api.py [hunk 2]
```diff
@@ -4,21 +4,26 @@
...
                 return expression
+        elif "," in piece:
+            splitter = piece.split(",")
+            try:
+                splitter = [int(p) for p in splitter]
+            except ValueError:
+                return expression
+            piece = tuple(splitter)
         pieces.append(piece)
-
```
Comment (adamchainz) [inline pretty_cron/api.py:15]: Don't gratuitously change lines in a PR that aren't to do with the functionality you're adding. It's just a stylistic preference, plus it makes your code harder to merge if say these lines get changed before your PR is looked at.

File: pretty_cron/api.py [hunk 3]
```diff
@@ -46,7 +51,7 @@ def _pretty_date(month_day, month, week_day):
             week_day_date = ""
 
         if month_day_date:
-            if month != "*":
+            if month != '*':
```
Comment (adamchainz) [inline pretty_cron/api.py:54]: This is especially unnecessary

**Comment by adamchainz** [comment]
Merged in 847170d9f70844ed88dfe89f39b768ba1ad98792 , thanks!

**Pull request closed without merging**